data/json/monsters/nether.json

```diff
@@ -740,7 +740,7 @@
     "harvest": "exempt",
     "special_attacks": [ [ "TENTACLE", 5 ] ],
     "death_function": [ "MELT" ],
-    "flags": [ "SMELLS", "HEARS", "NOHEAD", "BASHES", "GROUP_BASH", "SWIMS", "ATTACKMON", "PLASTIC", "ACIDPROOF", "NOGIB", "CLIMBS" ]
+    "flags": [ "SMELLS", "HEARS", "NOHEAD", "BASHES", "GROUP_BASH", "SWIMS", "PUSH_MON", "PLASTIC", "ACIDPROOF", "NOGIB", "CLIMBS" ]
```

Comment:
```suggestion
    "flags": [ "SMELLS", "HEARS", "NOHEAD", "BASHES", "GROUP_BASH", "SWIMS", "ATTACKMON", "PLASTIC", "ACIDPROOF", "NOGIB", "CLIMBS" ]
```

Reply:
Ill commit these suggestions when venera responds, just to be sure what the plan is, i am not opposed to there being monsters that are outwardly aggressive to anything they see, but i am fairly certain it can just be set up so that they instead have their own faction instead of the flag, although the flag might make things simpler for a single monster. Either way its a necessary tag for compatibility anyways.

Reply:
Their faction actually already causes them to be aggressive to almost anything (including each other), actually. The "almost" is for things like fish that they essentially just don't care about one way or the other.

Reply:
I agree.  These should not be Pushing.